### DISCUSSION TITLE
Update repository name

### DISCUSSION THREAD

**Discussion by shoebjoarder** [ude-soco/CourseMapper-webserver#1089] [General]
@ude-soco/developers The repo actually started with only the `webserver` but as the project matured, Angular and Python servers were later added. However, the repository name remained the same `CourseMapper-webserver`. 

Since the repository is now public, we need to think of having an official repository name for the community, so that we can refer to not only the official deployed link but also to the CourseMapper GitHub repository link in our future publications/presentations.  I am suggesting to update the repository name to either (1) `coursemapper-official` or (2) `CourseMapper`. If you have any other suggestions, please feel free to share it.

I am hoping that we will not be facing any major consequences in the deployment process, if we update it.

**Comment by qurat08** [comment]
I would vote for (1) but if possible with capitalization: 'CourseMapper-Official'
